protocol/claim_monero.go

```diff
@@ -87,3 +99,18 @@ func ClaimMonero(
 
 	return nil
 }
+
+// setSweepStatus sets the swap's status as `SweepingXMR` and writes it to the db.
+func setSweepStatus(info *swap.Info, sm SwapManager) error {
+	info.SetStatus(types.SweepingXMR)
+	err := sm.WriteSwapToDB(info)
+	if err != nil {
+		return err
+	}
+
+	if info.StatusCh() != nil {
```

Comment:
Are there code paths where this can be `nil` and everything is still okay? I'm wondering if there needs to be a logged message.

Reply:
it can be nil, for example if this is a restart of an ongoing swap, the `statusCh` will be nil